package.json

```diff
@@ -34,7 +34,7 @@
     "techdocs-cli": "node scripts/techdocs-cli.js",
     "techdocs-cli:dev": "cross-env TECHDOCS_CLI_DEV_MODE=true node scripts/techdocs-cli.js",
     "prepare": "husky install",
-    "postinstall": "husky install"
+    "postinstall": "husky install || true"
```

Comment:
This is needed because the root `package.json` has an install script to run husky, which is not installed when doing `--production` so the container fails to install. Looking for better solutions here, but not sure what would be a good one.

Reply:
Mm, I see. Hm this is probably OK